autodispose-rxlifecycle/src/main/java/com/ubercab/autodispose/rxlifecycle/package-info.java

```diff
@@ -15,7 +15,9 @@
  */
 
 /**
- * Interop for RxLifecycle.
+ * Interop for RxLifecycle. It could understand RxLifecycle's
```

Comment:
I think we can phrase this better, doesn't have to be too wordy. Let's do

> AutoDispose extensions for interop with RxLifecycle. This namely supports {@link LifecycleProvider}.

Reply:
Fixed - https://github.com/uber/AutoDispose/pull/118/commits/7f3e1a87bb905848c38251a68a79861612033919